_includes/setup-steps.html

```diff
@@ -1,13 +1,13 @@
 <section>
     <h3 class="c-subheading-2">Connecting to a network</h3>
     <blockquote class="c-blockquote">
-    <p class="c-paragraph">NOTE: Skip this step if you've already connected your device to a network in Step 1.</p>
+    <p class="c-paragraph">NOTE:  Skip this step if you've already connected your device to a network in Step 1.</p>
     </blockquote>
     <h4 class="c-subheading-4">Wired</h4>
     <p class="c-paragraph-4">If the device comes with [an Ethernet port | USB Ethernet adapter support] to enable a wired connection, attach an Ethernet cable to connect it to your network.</p>
     <h4 style="padding-top:36px;" class="c-subheading-4">Wireless</h4>
     <p class="c-paragraph-4">If the device supports Wi-Fi connectivity and you've connected a display to it, follow these instructions:</p>
-            1. In the IoT Core default application, click the <b>Settings</b> button next to the clock.<br>
+            1. In the Windows 10 IoT Core default application, click the <b>Settings</b> button next to the clock.<br>
```

Comment:
The app is called IotCoreDefault app. We don't have Windows 10 in the name.

Reply:
Ah, ok... I will make those changes.